commit 3fc001d0a2e2456fb266f689964fe116d58c2227
Author: nicm <nicm>
Date:   Tue Oct 27 09:15:21 2015 +0000

    Use copy-mode -et= in WheelUpPane binding, from Patrick Palka.

--- a/key-bindings.c
+++ b/key-bindings.c
@@ -227,7 +227,7 @@
 		"bind -n WheelUpStatus previous-window",
 		"bind -n MouseDrag1Pane if -Ft= '#{mouse_any_flag}' 'if -Ft= \"#{pane_in_mode}\" \"copy-mode -M\" \"send-keys -M\"' 'copy-mode -M'",
 		"bind -n MouseDown3Pane select-pane -mt=",
-		"bind -n WheelUpPane if-shell -Ft= '#{mouse_any_flag}' 'send-keys -M' 'if -Ft= \"#{pane_in_mode}\" \"send-keys -M\" \"copy-mode -e\"'",
+		"bind -n WheelUpPane if-shell -Ft= '#{mouse_any_flag}' 'send-keys -M' 'if -Ft= \"#{pane_in_mode}\" \"send-keys -M\" \"copy-mode -et=\"'",
 	};
 	u_int		 i;
 	struct cmd_list	*cmdlist;
